test/studies/arkouda/sub_test

```diff
@@ -102,7 +102,11 @@ if [ ${CHPL_TEST_ARKOUDA_STOP_AFTER_BUILD} = "false" ]; then
 
   # Run benchmarks
   if [ "${CHPL_TEST_ARKOUDA_PERF}" = "true" ] ; then
-    benchmark_opts="--gen-graphs --dat-dir $CHPL_TEST_PERF_DIR --graph-dir $CHPL_TEST_PERF_DIR/html"
+    benchmark_opts="--save-data --dat-dir $CHPL_TEST_PERF_DIR --graph-dir $CHPL_TEST_PERF_DIR/html"
+
+    if [ "${CHPL_TEST_SYNC_ARKOUDA_GRAPHS}" = "true" ] ; then
```

Comment:
Nit, but probably call this `GEN_ARKOUDA_GRAPHS` since it's disabling generation, not syncing

Reply:
Sorry, I actually meant `CHPL_TEST_GEN_ARKOUDA_GRAPHS` (must have missed the `CHPL_TEST_` prefix in my earlier haste)